src/protocolsupport/listeners/PlayerListener.java

```diff
@@ -76,10 +78,9 @@ public void onBlockPlace(BlockPlaceEvent event) {
 	@EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
 	public void onEntityDamage(EntityDamageEvent event) {
 		if((event.getCause() == DamageCause.FIRE_TICK || event.getCause() == DamageCause.FIRE)) {
-			for(Player p : Bukkit.getOnlinePlayers()) {
-				//Send extra status packets to all old connections within 48 blocks (2304 = 48^2).
-				if(p.getLocation().distanceSquared(event.getEntity().getLocation()) <= 2304) { 
-					Connection connection = ProtocolSupportAPI.getConnection(p);
+			for(Entity e : event.getEntity().getWorld().getNearbyEntities(event.getEntity().getLocation(), 48, 128, 48)) {
```

Comment:
@Shevchik Doesn't CraftBukkit call that server method using getNearbyEntities?
[Source](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/CraftWorld.java#706)